mora/service/address.py

```diff
@@ -256,3 +262,78 @@ def edit_address(employee_uuid, req):
     }
 
     c.bruger.update(payload, employee_uuid)
+
+
+@blueprint.route('/address_lookup/')
+@util.restrictargs('local', required=['query'])
+def get_geographical_addresses():
+    """Perform address lookup
+
+    .. :quickref: Address; Lookup
+
+    :queryparam str query: A query string to be used for lookup
+    :queryparam uuid local: An optional UUID of an organisation. If given, the
+        lookup will be inside the bounds of the organisation's municipality.
+
+    **Example Response**:
+
+    :<jsonarr uuid uuid: A UUID of a DAR address
+
+    .. sourcecode:: json
+
+      [
+        {
+          "location": {
+            "uuid": "397705eb-1752-4efc-8f1c-54fd0d76366c"
+          }
+        },
+        {
+          "location": {
+            "uuid": "37380cfe-3781-460e-95f0-36ad8798f5ea"
+          }
+        }
+      ]
+    """
+    return flask.jsonify(
```

Comment:
This seems a bit too much code to me — couldn't you just inline the functions below?